addons/udes_stock/models/procurement.py

```diff
@@ -39,3 +39,17 @@ def get_group(self, group_identifier, create=False):
                 _('Too many groups found for '
                   'identifier %s') % str(group_identifier))
         return results
+
+    @api.model
+    def run(self, product_id, product_qty, product_uom, location_id, name,
+            origin, values):
+        """ Override run to clean up empty pickings in case any refactoring has
+            occurred.
+        """
+        Picking = self.env['stock.picking']
+
+        super().run(product_id, product_qty, product_uom, location_id, name,
+                    origin, values)
+        Picking.unlink_empty()
+
+        return True
```

Comment:
should we return true, or should we return the result of the super call?

Reply:
The return of the super call is True.